app/assets/javascripts/ang/controllers/observation_search.js.erb

```diff
@@ -1280,6 +1280,7 @@ function( ObservationsFactory, PlacesFactory, TaxaFactory, shared, $scope, $root
       resetOnChange: false,
       bootstrapClear: true,
       searchExternal: false,
+      position: { my: "right top", at: "right bottom" },
```

Comment:
This just changes the position for everyone, including LTR users. You could do it for every individual autocomplete like you're doing here, but there's probably a way to do it once in generic_autocomplete.js.